file/ext4.lisp

```diff
@@ -575,24 +579,19 @@
                  (never (zerop block-n))
                  (follow-pointer disk superblock block-n fn indirection))))))
 
-(defun read-file (disk superblock bgdt inode-n)
-  (let ((blocks))
+(defun read-block-n (disk superblock bgdt inode block-n)
+  (let ((n -1))
     (do-file #'(lambda (block)
```

Comment:
It might be useful to split this into a function that takes a file block number and converts it to a disk block number, so clients don't need to iterate over every block in the file.